packages/bitcore-wallet-service/src/config.ts

```diff
@@ -72,7 +72,7 @@ module.exports = {
     defaultLanguage: 'en',
     defaultUnit: 'btc',
     subjectPrefix: '',
-    pushServerUrl: 'https://fcm.googleapis.com/fcm',
+    pushServerUrl: 'https://fcm.googleapis.com/v1/projects/bitpay-wallet/messages:send',
     authorizationKey: 'You_have_to_put_something_here'
```

Comment:
`authorizationKey `  is now deprecated right? Can you remove this and move the fcm_google_credentials here?